src/test/java/seedu/address/model/residence/NameTest.java

```diff
@@ -6,37 +6,37 @@
 
 import org.junit.jupiter.api.Test;
 
-import seedu.address.model.booking.Name;
```

Comment:
we should be testing Residence's Name here instead of TenantName

Reply:
yup fixed!